src/main/java/az/edu/turing/Entities/Family.java

```diff
@@ -88,8 +88,22 @@ public String toString() {
         return "Family{" +
                 "mother=" + mother.getName() + " " + mother.getSurname() +
                 ", father=" + father.getName() + " " + father.getSurname() +
-                ", children=" + Arrays.toString(children) +
+                ", children=" + childrenToString() +
                 ", pet=" + pet +
                 '}';
     }
+
+    private String childrenToString() {
```

Comment:
pls remove this method, it should use Humans toString method for print
looks like
`Family{father=Human{name='Lily', surname='Potter', year=1960, iq=0, schedule=null}, mother=Human{name='James', surname='Potter', year=1960, iq=0, schedule=null}, children=[Human{name='Harry', surname='Potter', year=1980, iq=97, schedule=null}, Human{name='Harry', surname='Potter', year=1980, iq=97, schedule=null}], pet=owl{nickname='Hedwig', age=7, trickLevel=49, habits=[eat, fly]}}`